features/FEATURE_LWIP/lwip-interface/lwip-sys/arch/lwip_sys_arch.c

```diff
@@ -237,7 +236,7 @@ err_t sys_mbox_trypost(sys_mbox_t *mbox, void *msg) {
  *                                  of milliseconds until received.
  *---------------------------------------------------------------------------*/
 u32_t sys_arch_mbox_fetch(sys_mbox_t *mbox, void **msg, u32_t timeout) {
-    uint32_t start = us_ticker_read();
+    uint32_t start = osKernelGetTickCount();
```

Comment:
isn't one tick 1millisecond thus this would return different time than us_ticker_read before?